### DISCUSSION TITLE
Allure Report: Multiple entry in History tab of Suites section

### DISCUSSION THREAD

**Discussion by swadeepnagar** [allure-framework/allure2#1680] [Bug Reports]
I am adding Allure Report in Selenium C#. Although successfully generated the report, but it shows multiple entries(for same execution) in History tab of Suites section. Find attached screenshot with highlighted issue.
<img width="880" alt="Allure_MultipleHistoryIssue" src="https://user-images.githubusercontent.com/103187564/171897011-baff59f4-b814-43fb-abbb-05cccbac9368.png">

**Comment by SohanTirpude** [comment]
Hello @swadeepnagar,

I am also facing the same issue for which I have created this ticket: https://github.com/allure-framework/allure2/issues/2402 as well, but so far no help.

Were you able to fix this issue by any how? Kindly let me know.

Thank you.
